doc/manual/src/SUMMARY.md.in

```diff
@@ -32,6 +32,7 @@
   - [Operators](language/operators.md)
   - [Derivations](language/derivations.md)
     - [Advanced Attributes](language/advanced-attributes.md)
+    - [Import From Derivation](language/import-from-derivation.md)
```

Comment:
```suggestion
    - [Reading build results](language/reading-build-results.md)
```

Reply:
Let us do the renaming in another pull request to focus on the essence of this one. See my [top-level comment](https://github.com/NixOS/nix/pull/7332#issuecomment-1324800375).